media/js/firefox/new/de-experiment.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```

Comment:
Small nit - should have `Mozilla` in the parens here (to name the variable being passed in at the bottom of the file).